app/test_split_integer.py

```diff
@@ -1,21 +1,91 @@
+import pytest
+
 from app.split_integer import split_integer
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value,number_of_parts,expected_result",
+    [
+        (8, 1, [8]),
+        (6, 2, [3, 3]),
+        (17, 4, [4, 4, 4, 5]),
+        (32, 6, [5, 5, 5, 5, 6, 6])
+    ],
+)
+def test_sum_of_the_parts_should_be_equal_to_value(
+        value: int,
+        number_of_parts: int,
+        expected_result: list[int, ...]
+) -> None:
+    assert split_integer(value, number_of_parts) == expected_result
 
 
-def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value,number_of_parts",
+    [
+        (8, 1),
+        (6, 2),
+        (17, 4),
+        (32, 6)
+    ],
+)
+def test_should_split_into_equal_parts_when_value_divisible_by_parts(
+        value: int,
+        number_of_parts: int
+) -> None:
+    assert (
+        split_integer(value, number_of_parts)
+        == [value // number_of_parts] * number_of_parts
+    )
 
 
-def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+@pytest.mark.parametrize(
+    "value,number_of_parts",
+    [
+        (8, 1),
+        (6, 2),
+        (17, 4),
+        (32, 6)
+    ],
+)
+def test_should_return_part_equals_to_value_when_split_into_one_part(
+        value: int,
+        number_of_parts: int
+) -> None:
+    assert split_integer(value, number_of_parts) == [value // number_of_parts]
```

Comment:
This test case seems to be redundant because it checks the same condition as the test case in lines 32-39. If the number of parts is 1, the value should indeed be split into one part, which is the value itself. This is already covered by the previous test case.